src/frog-base.tsx

```diff
@@ -380,19 +380,7 @@ export class FrogBase<
       for (const [key, value] of Object.entries(headers ?? {}))
         c.header(key, value)
 
-      const isDevEnabled =
-        // check if devtools are enabled on constructor.
-        (this.dev?.enabled ?? true) &&
-        // check if route has `/dev` path.
-        this.hono.routes.some((r) => {
-          const currentFullPath =
-            (this.basePath === '/' ? '' : this.basePath) + parsePath(path)
-          if (!r.path.startsWith(currentFullPath)) return false
-          if (!r.path.includes('/dev')) return false
-          return true
-        })
```

Comment:
Do we still need this for edge environments? Think `happy-dom` and `@farcaster/core` were the constraining packages in the devtools.

Reply:
going to drop those